packages/data-point-service/lib/cache-middleware.test.js

```diff
@@ -198,45 +202,36 @@ describe('setStaleWhileRevalidateEntry', () => {
 })
 
 describe('revalidateEntry', () => {
-  let logDebug
   let logError
   const cache = {
     ttl: 200,
     staleWhileRevalidateTtl: 400
   }
 
   beforeEach(() => {
-    const logger = winston.loggers.get('data-point-service')
-    logDebug = logger.debug
-    logError = logger.error
-    // to hide when not testing it got called
-    logger.debug = () => true
+    logError = console.error
+    mockDebug.mockReset()
   })
+
   afterEach(() => {
-    const logger = winston.loggers.get('data-point-service')
-    logger.debug = logDebug
-    logger.error = logError
+    console.error = logError
   })
 
   it('should log at debug level start and success', () => {
     const mocks = createMocks()
-    const logger = winston.loggers.get('data-point-service')
-    logger.debug = jest.fn()
-
     return CacheMiddleware.revalidateEntry(
       mocks.service,
       'key',
       cache,
       mocks.ctx
     ).then(() => {
-      expect(logger.debug.mock.calls).toMatchSnapshot()
+      expect(mockDebug.mock.calls).toMatchSnapshot()
     })
   })
 
   it('should log error level when error occurs', () => {
     const mocks = createMocks()
-    const logger = winston.loggers.get('data-point-service')
-    logger.error = jest.fn()
+    console.error = jest.fn()
```

Comment:
I wonder if it would be better to use jest.spyOn in cases like this instead of replacing and restoring console.error explicitly...

Reply:
im definitely not against it, my aim here was mainly to replace the Winston, we could do a pass to use better mocking practices